lcls-twincat-motion/Library/POUs/Motion/PMPS/FB_MiscStatesErrorFFO.TcPOU

```diff
@@ -97,7 +97,7 @@ ffUnknown(
 ftTripCount(CLK:=ffBeamParamsOk.i_xOK);
 
 IF ftTripCount.Q THEN
-    nTripCount := nTripCount + 1;
+    nTripCount := MAX(nTripCount + 1, 5);
```

Comment:
In retrospect, I think we might need to put this at a higher number than 5, else the debouncer is itself bouncy because it trips exactly at 5. Any thoughts?

Reply:
Alternatively, we can make it reset automatically at 0 instead of at 4

Reply:
Maybe I'll make a small PR to make it reset only at zero